basic_hashtable/b_hashtables.py

```diff
@@ -16,28 +16,45 @@ def __init__(self, key, value):
 class BasicHashTable:
     def __init__(self, capacity):
         self.capacity = capacity
-        self.count = 0
+        self.storage = [None] * capacity
         pass
 
 
 # '''
 # Fill this in.
 # Research and implement the djb2 hash function
 # '''
-def hash(string, max):
-    hashed = 5381
-    for i in string:
-        hashed = ((hashed << 0) + hashed) + ord(string) 
-    return hash & 0xFFFFFFFF
-
+def hash(string):
+    hash = 5381
 
+    for i in string:
+        hash = (((hash << 5) + hash) + ord(i)) & 0xFFFFFFFF
```

Comment:
The and operation should be in the return statement.

Reply:
Don't remember why we didn't need it .... It's been a while.